docs/devices-api/more-info/cordova.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Cordova"
+description: "A plugin to integrate with Metriport on Cordova - including Apple Health integrations."
+---
+
+<Info>
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
+</Info>
+
+## Installation
+
+```sh
+cordova plugin add https://github.com/metriport/cordova-sdk
+```
+
+You'll need to ensure that the MetriportSDK pod is installed as well.
+
+```sh
+cd platforms/ios
+pod install
+```
+
+## Usage
+
+To use our SDK go to your index.html add:
```

Comment:
nit: To use our SDK, go to your `index.html` and add:

Reply:
I see you marked these as resolved but I still see the original code, maybe forgot to push the changes?